src/converter.js

```diff
@@ -377,6 +393,61 @@ var resolveRef = function(oai, ref) {
   return recursive(oai, 1)
 }
 
+/**
+ * Returns the encoded value for defined content
+ *
+ * @param  {any} value The sampled value to encode
+ * @param  {string} contentType The content-type of the value
+ * @param  {object} content The content object used for further details
+ * @return {any}
+ */
+var encodeValue = function(value, contentType, content) {
+  if (content.encoding) {
+    const encodingKeys = Object.keys(content.encoding);
+    encodingKeys.map((encodingKey) => {
+      value[encodingKey] = encodeValue(value[encodingKey], content.encoding[encodingKey].contentType, content)
+    });
+  }
+
+  switch (contentType) {
+    case 'application/json':
+      return JSON.stringify(value);
+
+    case 'application/x-www-form-urlencoded':
+      return Object.keys(value).map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(value[key])}`).join('&');
+
+    case 'application/xml':
+      const xmlOptions = {
+        header: true,
+        indent: '  '
+      };
+      return toXML(value, xmlOptions);
+
+    case 'multipart/form-data':
+      let formData = new FormData();
+      Object.keys(value).forEach((key) => formData.append(key, value[key]));
+      return formData;
+
+    case 'application/octet-stream':
+    case 'text/plain':
+    case '*/*':
+      if (typeof value === 'object') {
+        value = JSON.stringify(value);
+      }
+      return Buffer.from(value).toString('base64');
+
+    case 'image/jpg':
+    case 'image/jpeg':
+      return Buffer.from('FF D8 FF DB').toString('base64');
```

Comment:
![image](https://user-images.githubusercontent.com/38690835/68779817-5282c280-0646-11ea-9fa3-c71de2bdc11b.png)
```suggestion
      return Buffer.from([0xFF, 0xD8, 0xFF, 0xDB], 'hex').toString('base64');
```
https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_array